test/integration/guest_creates_login_with_incorrect_creditials_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class GuestCreatesLoginWithIncorrectCreditialsTest < ActionDispatch::IntegrationTest
+  test "guest puts in incorrect password confirmation and gets error message" do
+    visit new_user_path
+    fill_in "First name", with: "Bruce"
+    fill_in "Last name", with: "Wayne"
+    fill_in "Username", with: "Batman"
+    fill_in "Password", with: "darkness"
+    fill_in "Password confirmation", with: "parents_dead"
+    click_on "Create Account"
+
+    assert_equal '/users', current_path
+    assert page.has_content? "Password confirmation doesn't match Password"
+  end
+
+  test 'guest puts in name that is already created' do
+    user = create(:user)
+
+    visit new_user_path
+    fill_in "First name", with: "Bruce"
+    fill_in "Last name", with: "Wayne"
+    fill_in "Username", with: "user"
+    fill_in "Password", with: "darkness"
+    fill_in "Password confirmation", with: "darkness"
+    click_on "Create Account"
+
+    assert_equal '/users', current_path
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.